catalog/geoscience-aus-cat/projects_geoscience-aus-cat_assets_geomedians_LS5.jsonnet

```diff
@@ -0,0 +1,179 @@
+local id = 'projects/geoscience-aus-cat/assets/geomedians_LS5';
+local subdir = 'geoscience-aus-cat';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local version = '3.1.0';
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  "gee:skip_indexing": true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  version: version,
+  title: 'DEA Geometric Median and Median Absolute Deviation - Landsat 5 ' + version,
+  'gee:type': ee_const.gee_type.image,
+  description: |||
+    This product provides statistical tools to exploit the time series of Earth observation data available in Digital Earth Australia, providing annual images of general conditions and how much an area changes for a given year.
+
+    For more information, please see the
+    [DEA Geometric Median and Median Absolute Deviation](https://cmi.ga.gov.au/data-products/dea/645/dea-geometric-median-and-median-absolute-deviation-landsat#basics)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference),
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.1109/TGRS.2017.2723896',
+      type: ee_const.media_type.html,
+    },
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.1109/IGARSS.2018.8518312',
+      type: ee_const.media_type.html,
+    },
+    {
+      rel: ee_const.rel.source,
+      href: 'https://data.dea.ga.gov.au/?prefix=derivative/ga_ls5t_nbart_gm_cyear_3/',
+    },
+  ],
+  keywords: [
+    'australia',
+    'ga',
+    'landsat-derived',
```

Comment:
Please use landsat_derived. For consistency, we only allow underscores in keywords.